config/dns/config.go

```diff
@@ -9,6 +9,20 @@ import (
 // Configure configures individual resources by adding custom
 // ResourceConfigurators.
 func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("google_dns_managed_zone", func(r *config.Resource) {
+		r.References["peering_config.target_network.network_url"] = config.Reference{
+			Type:              "github.com/upbound/provider-gcp/apis/compute/v1beta1.Network",
+			RefFieldName:      "NetworkRef",
+			SelectorFieldName: "NetworkSelector",
+			Extractor:         common.PathSelfLinkExtractor,
+		}
+		r.References["private_visibility_config.networks.network_url"] = config.Reference{
+			Type:              "github.com/upbound/provider-gcp/apis/compute/v1beta1.Network",
+			RefFieldName:      "NetworkRef",
+			SelectorFieldName: "NetworkSelector",
```

Comment:
While adding reference fields, we do not have to specify RefFieldName and SelectorFieldName. For this PR, adding them caused API changes. So removing these specifications will make us avoid API changes.

Reply:
Thx, done